src/Client.php

```diff
@@ -49,7 +49,8 @@ class Client extends Model
     public function user()
     {
         return $this->belongsTo(
-            config('auth.providers.'.config('auth.guards.api.provider').'.model')
+            config('auth.providers.'.config('auth.guards.api.provider').'.model'),
+            resolve(config('auth.providers.'.config('auth.guards.api.provider').'.model'))->getKeyName()
```

Comment:
A model doesn't needs to be resolved from the container so let's not use the resolve method here but do something like this instead:

```suggestion
           (new config('auth.providers.'.config('auth.guards.api.provider').'.model'))->getKeyName()
```

Reply:
I can't do `new` on a `config()`helper one solution could be to save the config() in a variable and then make it like this:

````
    public function user()
    {
        $class = config('auth.providers.' . config('auth.guards.api.provider') . '.model');

        return $this->belongsTo($class, (new $class)->getKeyName());
    }
````  

or i could store it in a member variable like in the Passport.php class.

````
    protected $userModel = config('auth.providers.' . config('auth.guards.api.provider') . '.model');
````

Reply:
Ah yeah, sorry, my bad 🤦‍♂ 

The first option is the better solution I think.